salt/iiif/config/etc-loris2-loris2.conf

```diff
@@ -16,109 +16,92 @@
 #
 # <http://www.voidspace.org.uk/python/configobj.html#unrepr-mode>
 #
-# String interpolation is enabled using the "template" style. OS environment
-# variables are available for interpolation, e.g., run_as_user='$USER'
-#
-# <http://www.voidspace.org.uk/python/configobj.html#string-interpolation>
+# String interpolation is disabled.
 #
 
 [loris.Loris]
-    tmp_dp = '{{ pillar.iiif.loris.storage }}/tmp'
-    www_dp = '/var/www/loris2'
-    run_as_user = 'loris'
-    run_as_group = 'loris'
-    enable_caching = True
-    redirect_canonical_image_request = False
-    redirect_id_slash_to_info = True
-    
-    # max_size_above_full restricts interpolation of images on the server.
-    # Default value 200 means that a user cannot request image sizes greater than
-    # 200% of original image size (width or height).
-    # Set this value to 100 to disallow interpolation. Set to 0 to remove
-    # size restriction.
-    max_size_above_full = 200
-
-#proxy_path=''
-# cors_regex = ''
-# NOTE: If supplied, cors_regex is passed to re.search():
-#    https://docs.python.org/2/library/re.html#re.search
-# Any url_root:
-#    http://werkzeug.pocoo.org/docs/latest/wrappers/#werkzeug.wrappers.BaseRequest.url_root
-# (i.e., https?://domain.edu(:port)?/) that matches will be
-# set to the value of Access-Control-Allow-Origin.
+tmp_dp = '{{ pillar.iiif.loris.storage }}/tmp' # r--
+www_dp = '/var/www/loris2' # r-x
+run_as_user = 'loris'
+run_as_group = 'loris'
+enable_caching = True
+redirect_canonical_image_request = False
+redirect_id_slash_to_info = True
 
 [logging]
-    log_to = 'console'
-    log_level = 'INFO'
-    #log_dir = '/var/log/loris2'
-    #max_size = 5242880
-    #max_backups = 5
-    format = '%(asctime)s (%(name)s) [%(levelname)s]: %(message)s'
+log_to = 'console'    # 'console'|'file'
+log_level = 'INFO' # 'DEBUG'|'INFO'|'WARNING'|'ERROR'|'CRITICAL'
+#log_dir = '/var/log/loris2' # rw-
+#max_size = 5242880 # 5 MB
+#max_backups = 5
+format = '%(asctime)s (%(name)s) [%(levelname)s]: %(message)s'
 
 [resolver]
 {% for key, value in pillar.iiif.loris.resolver.items() %}
-    {{ key }} = '{{ value }}'
+{{ key }} = '{{ value }}'
+{% endfor %}
+# only used by HTTP resolvers
+cache_root = '{{ pillar.iiif.loris.storage }}/cache-resolver'
```

Comment:
This path also needs to change to adapt to the container (at that point it will be hardcoded rather than configurable via pillar).

Should still be mounted as a volume as there is machinery on the host to clean it only when full, and that would still survive container restarts. I'd consider bad practice to have containers write lots of stuff in their temporary file system (GBs in this case) as from the host you see `/var/lib/docker/volumes/bla-bla-bla-some-id` occupying more and more space without any insight on what that is.

Reply:
I've taken these paths in the config and reverted them back to their original loris defaults and bound the variable/pillar-derived paths to the directory mounts:

https://github.com/elifesciences/iiif-formula/pull/18/files#diff-1d485fdd35aed6965f26828f3486ff05R136-R139

the idea is to not deviate very far from the default configuration (v3 of loris is due soon apparently)